src/stream/src/executor/sink.rs

```diff
@@ -294,6 +298,7 @@ impl<F: LogStoreFactory> SinkExecutor<F> {
         mut log_writer: W,
         actor_id: ActorId,
         rate_limit_tx: UnboundedSender<RateLimit>,
+        update_config_tx: UnboundedSender<HashMap<String, String>>,
```

Comment:
Given that we dispatch full settings on the connector, how about reusing `rebuild_sink_tx`